frontend2/src/views/CommonIssues.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import {
+  defaultCommonIssuesText,
+  commonIssuesText,
+} from "../content/ManageContent";
+
+import { useEpisodeId } from "../contexts/EpisodeContext";
+import Markdown from "../components/elements/Markdown";
+import SectionCard from "../components/SectionCard";
+import { CommonIssuesPage } from "../content/ContentStruct";
+
+const DebuggingTips = (): JSX.Element => {
+  const { episodeId } = useEpisodeId();
+  const currentCommonIssuesText =
+    commonIssuesText[episodeId] ?? defaultCommonIssuesText;
+  return (
+    <div className="flex h-full w-full flex-col overflow-y-auto bg-white p-6">
+      <div className="flex flex-1 flex-col gap-8">
+        <SectionCard>
```

Comment:
Would it be better to move the section title to the `title` prop of `SectionCard` instead of in the markdown? I think it could be.